doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc

```diff
@@ -40,6 +40,7 @@ To use a Git hooks directory, make the following changes:
 . If you selected RH-SSO authentication, configure RH-SSO for {CENTRAL}:
 .. Enter the client name in the *Client name* field and the client secret in the *Client secret* field. If a client with this name does not exist, the deployment attempts to create a new client with this name and secret.
 .. If the deployment is to create a new client, enter the HTTP and HTTPS URLs that will be used for accessing {CENTRAL} into the *SSO HTTP URL* and *SSO HTTPS URL* fields. This information is recorded in the client.
+. Optional: If you are configuring a high-availability environment, set the user name and password for the DataGrid component in the *DataGrid username* and *DataGrid password* fields. By default, the user name is `infinispan` and the password is generated automatically. 
```

Comment:
Please use "username" instead of  "user name", in two places here

Reply:
Can't. Style guide.